djstripe/models.py

```diff
@@ -380,7 +389,9 @@ def _sync_subscriptions(self, **kwargs):
 
 @class_doc_inherit
 class Event(StripeEvent):
-    __doc__ = getattr(StripeEvent, "__doc__")
+    """A model extending StripeEvent providing dj-stripe functionality."""
```

Comment:
Again, we don't need these because the documentation is inherited and users don't see docs for StripeObjects